### PR TITLE
Render documentation social cards; other cleanups

### PR DESCRIPTION
**Pull request by gordonhart**
Fixes KOL-2503
Fixes KOL-2505
Fixes KOL-2506
Fixes KOL-2514
Fixes KOL-2515
Fixes KOL-2512
Fixes KOL-2519
Fixes KOL-2517

### CODE REVIEW

File: mkdocs.insiders.yml [hunk 1]
```diff
@@ -26,6 +26,15 @@ theme:
...
 plugins:
+  social:
+    # cache: false
+    # debug: true
+    cards_layout: custom
+    cards_layout_dir: docs/layouts
+    # cards_layout_options:
+    #   font_family: Inter
+    #   background_color: "#6434c1"
+    #   background_image: docs/assets/images/background.svg  # TODO: not properly applied -- debug
```
Comment (ly-kolena) [inline mkdocs.insiders.yml]: Are the comments here intentional to be kept?

File: mkdocs.yml [hunk 2]
```diff
@@ -52,7 +52,8 @@ nav:
               - Base Definitions: reference/legacy/built-in/base.md
   - Help & FAQ:
     - Help & FAQ: faq/index.md
-  - Sign in: https://app.kolena.io
+  - Sign in:
+    - Sign in: sign-in/index.md
```
Comment (ly-kolena) [inline mkdocs.yml]: What's the benefit of this alternative route?

Side note: previous implementation works for me, but new one doesn't until I hit reload button. This is from me running the server locally.

File: mkdocs.insiders.yml (hunk 1 above)
Comment (gordonhart) [reply mkdocs.insiders.yml]: No, thanks. Removed

File: mkdocs.yml (hunk 2 above)
Comment (gordonhart) [reply mkdocs.yml]: This is a kludge to render a link next to the `Sign in` button. It was working originally but now does not seem to work for me either. Might have to rip out and lose the external link icon...

Comment (gordonhart) [reply mkdocs.yml]: Removed in favor of a more subtle arrow character.

**Review by ly-kolena — APPROVED** [review]

**Pull request merged**